docs/python/settings-reference.md

```diff
@@ -18,6 +18,7 @@ Refer to [User and workspace settings](/docs/getstarted/settings.md) to find our
 
 | Setting | Default | Description |
 | --- | --- | --- |
+| python.condaPath | `"conda"` | Path to the conda command, or the path to a folder containg the Conda interpreter. |
```

Comment:
"Path to the `conda` executable."

Basically the folder isn't a valid option and there's a typo for "containg".